### PR TITLE
Create portfolio projects

### PR DESCRIPTION
**Pull request by nicupop729**
## Portfolio Project

For this branch, I have worked on the following:
- create HTML for the Work Experience section and the Knowledge - Skills section;
- add CSS for the same sections;

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,98 +12,264 @@
...
+        </p>
+        <a href="#" class="button b7">Get my Resume</a>
       </div>
     </section>
-
+    <section class="experience-section">
+      <ul class="experience-ul">
+        <li class="experience">
+          <div class="logo">
+            <img
```
Comment (safafa) [inline index.html]: - [x] The technology categories ("Languages", "Frameworks" and "Skills") list belong to the about me section. Kindly move it there.

File: src/style.css [hunk 2]
```diff
@@ -7,10 +7,17 @@
 }
 
 body {
+  width: 97vw;
+}
+
```
Comment (safafa) [inline src/style.css]: image

- [x] As you can see in the picture the horizontal scroll bar is showing for mobile view under 432px. Kindly fix the issue.
image
original design:
image

- [ ] As you can see in the picture your project looks a bit different from the original design in mobile view. Kindly fix the issue.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @nicupop729  You did a great job so far 👏🏼 👏🏼 

- Commit messages are professional
- The pull request is descriptive
- You did a good job in following the design
- Well done on using HTML semantic tags

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @nicupop729 , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**